### ISSUE TITLE
Malformed input data creates bad graphics

### ISSUE THREAD

**Issue by erluko** [erluko/flowgridviz#41]
Issues like unpareseable IP addresses or port numbers map data to hash(0). Those data points, when clicked, yield useless information.

Throw an error on bad data.

Imported from Georgia Tech github:
```
{"created_at":"2019-04-18T10:57:47Z","updated_at":"2019-04-18T11:11:21Z","closed_at":"2019-04-18T11:11:20Z","html_url":"https://github.gatech.edu/ekobrin3/2019-Spring-CSE6242-flowgridviz/issues/41","id":78865}
```

**Comment by erluko** [comment]
Fixed in: 9e595fbb3db240366b36b8ac19ddb58c4517c9f3
Further cleanup to the loading pages, etc. happened later.

Imported from Georgia Tech github:
```
{"created_at":"2019-04-18T10:58:25Z","updated_at":"2019-04-18T10:58:25Z","html_url":"https://github.gatech.edu/ekobrin3/2019-Spring-CSE6242-flowgridviz/issues/41#issuecomment-42125","id":42125}
```

**Comment by erluko** [comment]
Also relevant: 643373c953b7b3a0b99ae6970ff654bec09eb07b which skips header rows rather than triggering an error or undefined state.


Imported from Georgia Tech github:
```
{"created_at":"2019-04-18T11:11:20Z","updated_at":"2019-04-18T11:11:20Z","html_url":"https://github.gatech.edu/ekobrin3/2019-Spring-CSE6242-flowgridviz/issues/41#issuecomment-42129","id":42129}
```
